docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc

```diff
@@ -52,6 +52,9 @@ You can clone the Git repository by running the command `git clone {quickstarts-
 
 The solution is located in the `security-openid-connect-quickstart` link:{quickstarts-tree-url}/security-openid-connect-quickstart[directory].
 
+:sectnums:
```

Comment:
The numbering scheme is unusual, I know some other OIDC and security tutorials still use it, but these docs are in minority, simple TOC works fine, as far as I recall I removed it based on the feedback from QE, I'd rather keep removing the numbers in other docs instead

Reply:
@sheilamjones FYI, the PR which goes to 3.6 will also remove the numbering scheme, so IMHO it is ok without reintroducing it

Reply:
Hi @sberyozkin, I saw that alright that the numbering was removed in 3.6. I think it was previously removed (or after feedback from QE) because there was an issue with the numbering itself. It wasn't rendering correctly and was rendering with a dot before each number as well as after, that is,  ".1", ".2" etc. 
We fixed this in this latest version on Main, by removing the extra subheading "Procedure".  IMHO, I think the numbering does help to differentiate each individual step of the tutorial. Perhaps, I could remove for now and revisit it again in the future.

Reply:
@sheilamjones OK, sure, lets have a wider discussion, I kind of like a non-numbering scheme, but appreciate you like the alternative, so I will support the numbering one if we have a wide enough consensus :-)

Reply:
Sounds good. I removed the numbering from here for now, but think it would be good to revisit this collectively and have a wider consensus on numbering in the tutorials. Many thanks.